pllug-presentation-system/presentation-data/domdocumentdivider.h

```diff
@@ -0,0 +1,32 @@
+#ifndef DOMDOCUMENTDIVIDER_H
+#define DOMDOCUMENTDIVIDER_H
+
+#include <QStringList>
```

Comment:
Try to include header files in *.cpp, if they not 100% needed in *.h file. You may read ["C++ Header File Guidelines"](http://umich.edu/~eecs381/handouts/CppHeaderFileGuidelines.pdf) for more information.